apps/fishing-map/features/map/map-layers.hooks.ts

```diff
@@ -25,3 +26,10 @@ export const useMapDeckLayers = () => {
 
   return layers
 }
+
+export const useMapVesselLayer = (dataview: UrlDataviewInstance) => {
```

Comment:
🤔 my first thought is that this hook might belong to layer hooks and not map hooks. 
It is true that consuming a `UrlDataviewInstance` is coupling it with the app, but maybe it can consume an `id:string` and be used from the outside as `useMapVesselLayer(dataview.id)`, that way it might be decoupled from the workspace-dataview-dataviewinstance flow and be reused in other apps that do not rely on that? If this makes any sense 😅

Reply:
I like the idea, implemented [here](https://github.com/GlobalFishingWatch/frontend/commit/13cbf713005433d8d538dd1ee3b108f475b02094)